src/_modules/footer/footer.scss

```diff
@@ -1,3 +1,112 @@
 .footer {
-    
+  background: transparentize($mainColor, 0.9);
+
+  &__contact {
+    justify-content: center;
+
+    &__container {
+      display: flex;
+      flex-wrap: wrap;
+      transform: translateY(-30%);
+      background: $mainColor;
+      width: 80%;
+      max-width: 900px;
+      margin-left: auto;
+      margin-right: auto;
+      justify-content: center;
+      padding: 8px;
+      margin-bottom: -20px;
+
+      @include mq(md) {
+        flex-wrap: nowrap;
+      transform: translateY(-50%);
+
+      }
+
+      &__data {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+
+        @include mq(md) {
+          flex-wrap: nowrap;
+        }
+
+        i {
+          order: 2;
+          color: $white;
+          font-size: 24px;
+          align-self: center;
+          padding: 0 10px;
```

Comment:
separate the include from the style rules

Reply:
Done